### ISSUE TITLE
GMFCS level I

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#7278]
**Preferred term label:**
GMFCS level I

**Synonyms**
The Gross Motor Function Classification System level I

**Definition (free text, please give PubMed ID)**
Children with cerebral palsy classified under GMFCS level I can walk at home, school, outdoors and in the community, climb stairs without the use of a railing; they can perform gross motor skills such as running and jumping, but speed, balance and coordination are limited (PMID: 27188686).

**Parent term (use hpo.jax.org/app)**
Functional motor deficit HP:0004302

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

@pnrobinson

**Comment by pnrobinson** [comment]
As mentioned, I think these terms are not good for HPO because they bundle a number of different items. For HPO, we want to capture the detailed data.
